extension-manifest-v3/scripts/build.js

```diff
@@ -123,6 +124,43 @@ engines.forEach((engine) => {
   if (result.stderr) process.exit(1);
 });
 
+// extract resources
+shelljs.mkdir('-p', resolve(options.outDir, 'web_accessible_resources'));
+
+const seenWebAccessibleResources = new Set();
+
+FiltersEngine.deserialize(
```

Comment:
this procedure could be done in the engine download script. That way it wont have be re-created on every build, but only when the engine is changed

Reply:
Moved.